contrib/pipelines/core/Makefile

```diff
@@ -0,0 +1,20 @@
+SRCS := \
```

Comment:
Move Makefile outside so that it sits in skydive/contrib/pipelines/ (and not inside the code folder)

Reply:
Add README.md file that explains what is the exporter module used for 
(the text should reference to internal README.md files for the specific exporters)
(the text should explain how to build a new exporter (ref to dummy pipeline)

Reply:
Want to leave each with an independent makefile, while each have: `all`, `clean` and `run` targets - not sure it makes sense to attempt eliminating this minor duplication.

Added minimal readme to secadvisor, regarding text for building a pipeline, as the pipeline abstraction is still changing and expanding to meet needs of AWS Flow Logs and prefer to postpone this.